.eslintrc.json

```diff
@@ -136,6 +136,14 @@
                 "react/jsx-key": "off",
                 "no-console": "off"
             }
+        },
+        {
+            "files": ["packages/react-integration/demo-app-ts/**/*"],
+            "rules": {
+                "patternfly-react/no-anonymous-functions": "off",
+                "react/react-in-jsx-scope": "off",
```

Comment:
There is no need for this, as Vite uses the [JSX transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).